<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertActiveForm</class>
 <widget class="QDialog" name="InsertActiveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление записи в активный список</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выберите фамилию сотрудника:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Выберите банк:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Введите сумму:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
